modules/dragster/src/test/ts/browser/datatransfer/DataTransferTest.ts

```diff
@@ -222,8 +223,9 @@ describe('browser.dragster.datatransfer.DataTransferTest', () => {
       assert.strictEqual(transfer.files.length, 2, 'Should have same number of files');
 
       transfer.clearData();
-      if (isFirefox || isSafari) {
+      if (isFirefox || isSafari || isChrome) {
```

Comment:
This is all browsers that we test now so I don't think we need the `if` check anymore?

Reply:
different browser still have different behaviours, mainly recorded in those two tickets
https://ephocks.atlassian.net/browse/TINY-10743
https://ephocks.atlassian.net/browse/TINY-10742